eureka-webapp/src/main/webapp/assets/js/eureka.job.js

```diff
@@ -46,6 +46,15 @@ window.eureka.job = new function () {
 						$('#destinationConfig').text(data.destinationId);
 						$('#startedDate').text(data.startedDateFormatted);
 						$('#finishedDate').text(data.finishedDateFormatted);
+                                                if(data.status === "Completed" && data.getStatisticsSupported){
+                                                    var browseOutputLink='';
+                                                    browseOutputLink = '<a href="'+$('#getStatisticsSupported').attr('contextPath')+'/protected/jobstats?jobId='+data.jobId+'">Browse Output</a>';                                                 
+                                                    $('#getStatisticsSupported').html(browseOutputLink).show();
+
+                                                }else{
+                                                    $('#getStatisticsSupported').hide();
+                                                }
+                                                
```

Comment:
no, it only takes care of browse output link, not configured link(s). Because when I tested reloading the job page on local with H2 when a job is completed, I can only see the browse output link, so I assumed that the link that needs to be shown when job is completed is browse output link.

One thing is, the configured link(s) always returns a null List on local with H2, so even with working front-end code, it will not show. I think it is because we do not have i2b2 back-end, and oracle/postgreSQL database on local. So it will be hard to test.

I will modify the code for configured link(s), but may not be able to test until you release a new eureka-package with my code changes, then I can test on postgreSQL and oracle VMs.